.github/workflows/release.yaml

```diff
@@ -1,18 +1,18 @@
 name: release
```

Comment:
this file is supposed to be responsible for releasing the action-release-workflows repo. not a reusable workflow. would probably be good to add a comment to make that clearer